From 56cc24999bdf9096607b4a8ab188ec81da1857f1 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 15 Nov 2020 21:48:01 +0200
Subject: [PATCH] [SW-14676] fwupd: fix command print message prefix

The correct command prefix is "FW UPD", not "HL UPD".
Fix the messages prefix.

Change-Id: I4e18358f75a707e7eb7566859c3a17ee048073a3
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot/module_fwupd.c b/preboot/module_fwupd.c
index 1b3e632..3d79b66 100644
--- a/preboot/module_fwupd.c
+++ b/preboot/module_fwupd.c
@@ -146,14 +146,14 @@
 		break;
 	case HL_UPD_RST_STATE:
 		module_fwupd_reset_state();
-		preboot_puts("HL UPD RST STATE received\n");
+		preboot_puts("FW UPD RST STATE received\n");
 		break;
 	case HL_UPD_MAP_SRAM:
-		preboot_puts("HL UPD MAP SRAM received\n");
+		preboot_puts("FW UPD MAP SRAM received\n");
 		module_fwupd_sram_map(cmd.size, cmd.cmd);
 		break;
 	case HL_UPD_RUN_SRAM:
-		preboot_puts("HL UPD RUN SRAM received\n");
+		preboot_puts("FW UPD RUN SRAM received\n");
 		module_fwupd_sram_run();
 		break;
 	default:
